src/pages/home/report/ReportActionCompose/ComposerWithSuggestions/ComposerWithSuggestions.js

```diff
@@ -220,26 +222,45 @@ function ComposerWithSuggestions({
                     debouncedUpdateFrequentlyUsedEmojis();
                 }
             }
-            const newCommentConverted = convertToLTRForComposer(newComment);
+
+            let newCommentConverted = newComment;
```

Comment:
I suggest renaming this variable to `newCommentConvertedToLTR` since "converted" is ambiguous.

Reply:
I can change it, but it is the original name:

https://github.com/Expensify/App/blob/2feaac80198481546ee6aa630aa1c7b386ec5605/src/pages/home/report/ReportActionCompose/ComposerWithSuggestions/ComposerWithSuggestions.js#L223

Please confirm 🙏🏼

Reply:
Yes, please change it. Thanks for confirming.

Reply:
Done!